qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -209,48 +226,66 @@ DEV=
 rbd bench ${IMAGE} --io-type write --io-size=1024 --io-total=1024
 
 # unmap by image name test
-DEV=`_sudo rbd-nbd map ${POOL}/${IMAGE}`
+DEV=`_sudo rbd device --device-type nbd map ${POOL}/${IMAGE}`
 get_pid
 unmap_device ${IMAGE} ${PID}
 DEV=
 
 # map/unmap snap test
 rbd snap create ${POOL}/${IMAGE}@snap
-DEV=`_sudo rbd-nbd map ${POOL}/${IMAGE}@snap`
+DEV=`_sudo rbd device --device-type nbd map ${POOL}/${IMAGE}@snap`
 get_pid
 unmap_device "${IMAGE}@snap" ${PID}
 DEV=
 
 # map/unmap namespace test
 rbd snap create ${POOL}/${NS}/${IMAGE}@snap
-DEV=`_sudo rbd-nbd map ${POOL}/${NS}/${IMAGE}@snap`
-get_pid ${NS}
+DEV=`_sudo rbd device --device-type nbd map ${POOL}/${NS}/${IMAGE}@snap`
+get_pid ${POOL} ${NS}
 unmap_device "${POOL}/${NS}/${IMAGE}@snap" ${PID}
 DEV=
 
 # unmap by image name test 2
-DEV=`_sudo rbd-nbd map ${POOL}/${IMAGE}`
+DEV=`_sudo rbd device --device-type nbd map ${POOL}/${IMAGE}`
 get_pid
 pid=$PID
-DEV=`_sudo rbd-nbd map ${POOL}/${NS}/${IMAGE}`
-get_pid ${NS}
+DEV=`_sudo rbd device --device-type nbd map ${POOL}/${NS}/${IMAGE}`
+get_pid ${POOL} ${NS}
 unmap_device ${POOL}/${NS}/${IMAGE} ${PID}
 DEV=
 unmap_device ${POOL}/${IMAGE} ${pid}
 
+# map/unmap test with just image name
+DEV=`_sudo rbd device --device-type nbd map ${IMAGE}`
+get_pid
+unmap_device ${IMAGE} ${PID}
+DEV=
+
+# map/unmap test after changing default pool
+rbd config global set global rbd_default_pool ${ANOTHER_POOL}
+rbd config global get global rbd_default_pool | grep "^${ANOTHER_POOL}\\b"
+rbd create --size 10M ${IMAGE}
+DEV=`_sudo rbd device --device-type nbd map ${IMAGE}`
+get_pid ${ANOTHER_POOL}
+unmap_device ${ANOTHER_POOL}/${IMAGE} ${PID}
```

Comment:
Pass just `${IMAGE}`, let `rbd device unmap` figure out the default pool on its own.